superset-frontend/src/chart/ChartRenderer.jsx

```diff
@@ -228,6 +231,7 @@ class ChartRenderer extends React.Component {
         formData={formData}
         ownState={ownState}
         filterState={filterState}
+        extraFormData={extraFormData}
```

Comment:
I'm not sure that we need to pass `extraFormData` to `SuperChart` because this data should be used only in `buildQuery` function and `SuperChart` should use `filterState` for it needs @villebro thoughts?

Reply:
I would like to refer to `interactive_drilldown` in `extraFormData`, so I expect to put this property in properties.

Reply:
`interactive_drilldown` should be merged from `ExtraFormData` to `QueryFormData` when calling the `setDataMask` hook, and should really only be used when broadcasting to another chart. When a chart wants to broadcast data to itself only you should use `ownState`. Is this what you want to do?